README.md

```diff
@@ -15,9 +15,10 @@ This project is currently in development and is a first draft aimed at testing t
 2. Navigate to the project folder: `cd cylander`
 3. Install dependencies: `npm install`
 4. Generate scenes for development: `npm run generate:dev:all` (this add debug capability)
-5. Start the server: `npm run start` or Run watcher: `npm run watch` (regenerate dev scenes)
-6. Generate scenes for production: `npm run generate:prod:all`
-7. Run tests: `npm run test`
+5. Generate translation files: `npm run generate:translations`
```

Comment:
I think we should not have a separated task for translation, in my opinion this should be part of generating the scenes